docs/bundles/plain.md

```diff
@@ -2,43 +2,147 @@
+  valid `plain+v0` bundle that the provisioner can unpack.
+* The manifests directory should be flat: all manifests should be at the top-level with no subdirectories.
+* It is required that `kubectl apply` is able to process all .yaml files in the directory that make up a plain bundle. For example, multi-object YAML files are acceptable, but Ansible playbooks would not be.
+
+## Building a plain bundle
+### Prerequisites
 
-For example, below is an example of a file tree in a `plain+v0` bundle. It must have a `manifests` directory contains
-the Kubernetes resources required to deploy an application.
+In order to create a plain bundle for Rukpak, ensure your Kubernetes manifests are in a flat directory at the root of
+your project called `manifests/`. This allows the contents to be sourced and unpacked by the
+[plain provisioner](..provisioners/plain.md). It should look similar to:
 
-```tree
+```bash
+$ tree manifests
 manifests
 ├── namespace.yaml
+├── service_account.yaml
 ├── cluster_role.yaml
-├── role.yaml
-├── serviceaccount.yaml
 ├── cluster_role_binding.yaml
-├── role_binding.yaml
 └── deployment.yaml
 ```
 
 > Note: there must be at least one resource in the manifests directory in order for the bundle to be a valid
-> plain+v0 bundle.
+> `plain+v0` bundle.
 
-## Technical Details
+If you are using [kustomize](https://kustomize.io/) for building your manifests from a template, redirect the output into a single file under the `manifests/` directory. For example:
+```bash
+$ tree templates
+templates
+├── namespace.yaml
+├── service_account.yaml
+├── cluster_role.yaml
+├── cluster_role_binding.yaml
+├── deployment.yaml.yaml
+└── kustomization.yaml
+```
 
-* The static manifests must be located in the /manifests directory for the bundle to be a
-valid `plain+v0` bundle that the provisioner can unpack. A plain bundle image without a /manifests directory is
-invalid and will not be successfully unpacked onto the cluster.
-* The manifests directory should be flat: all manifests should be at the top-level with no subdirectories.
-* Including any content in the /manifests directory of a plain bundle that is not static manifests will result in
-a failure when creating content on-cluster from that bundle. Essentially, any file that would not
-successfully `kubectl apply` will result in an error, but multi-object YAML files, or JSON files, are fine. There will
-be validation tooling provided that can determine whether a given artifact is a valid bundle.
\ No newline at end of file
+```bash
+kustomize build templates > manifests/manifests.yaml
+```
+
+### Git source
+
+For using the bundle with a [`git` source](../sources/git.md), commit your `manifests/` directory to your `git` repository.
+
+### HTTP source
+
+For using the bundle with a [HTTP source](../sources/http.md), create a `.tgz` file that contains the `manifests/` directory and it's manifests. This `.tgz` file should be served when hitting the HTTP endpoint.
```

Comment:
Nitpick: "it's manifests" -> "its manifests"

Reply:
https://github.com/operator-framework/rukpak/pull/642/commits/902da450e601fa3c1b012ca90b21745fb1ecaa25